src/error/specification

```diff
@@ -1324,3 +1324,17 @@ severity:error
 ingroup:plugin
 module:path
 macro:INVALID_PERMISSION
+
+number:211
+description:error while writing file
+severity:error
+ingroup:plugin
+module:storage
+macro:WRITE_FAILED
+
+number:212
+description:an error occurred in the specload plugin (see reason)
```

Comment:
This seems to be a very specific (only for specload plugin's errors) and generic (various different errors as stated in the reason) at the same time.

@Piankero What do you think?

Reply:
Because we are changing the error concept, I didn't want to introduce additional errors that had to be removed/changed anyways.

Reply:
Thank you @kodebach. I agree with you